modules/op-release-notes-1-9.adoc

```diff
@@ -236,3 +236,119 @@ If you do not add any configuration data, you can use the default data in the AP
+
+* With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.
+
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. You can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+[id="deprecated-features-1-9_{context}"]
+== Deprecated and removed features
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, `ClusterTasks` are deprecated and planned to be removed in a future release. As an alternative, you can use `Cluster Resolver`.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the use of the `triggers` and the `namespaceSelector` fields in a single `EventListener` specification is deprecated and planned to be removed in a future release. You can use these fields in different `EventListener` specifications successfully.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the `tkn pipelinerun describe` command does not display timeouts for the `PipelineRun` resource.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the the `PipelineResource` custom resource (CR) is deprecated. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, custom image parameters from cluster tasks are deprecated. As an alternative, you can copy a cluster task and use your custom image in it.
+
+[id="known-issues-1-9_{context}"]
+== Known issues
+
+// .Operator
+
+* The `chains-secret` and `chains-config` config maps are removed after you uninstall the Red Hat OpenShift Pipelines Operator. As they contain user data, they should be preserved and not deleted.
+// https://issues.redhat.com/browse/SRVKP-2396
+
+// .PAC
+
+* When running the `tkn pac` set of commands on Windows, you may receive the following error message: `Command finished with error: not supported by Windows.`
++
+
+Workaround: Set the `NO_COLOR` environment variable to `true`.
+// https://issues.redhat.com/browse/SRVKP-2197
+
+* Running the `tkn pac resolve -f <filename> | oc create -f` command may not provide expected results, if the `tkn pac resolve` command uses a templated parameter value to function.
++
+Workaround: To mitigate this issue, save the output of `tkn pac resolve` in a temporary file by running the `tkn pac resolve -f <filename> > tempfile.yaml` command and then run the `oc create -f tempfile.yaml` command. For example, `tkn pac resolve -f <filename> -o /tmp/pull-request-resolved.yaml && kubectl create -f /tmp/pull-request-resolved.yaml`.
```

Comment:
`tkn pac resolve -f <filename> > tempfile.yaml` does not work, we need to change it to `tkn pac resolve -f <filename> -o tempfile.yaml`

Reply:
@VeereshAradhya - Hi, Thanks for looking at my PR. I hope you have looked at it thoroughly and this is the only comment you have. I hope there aren't any other comments which I need to incorporate. Please confirm.

Reply:
Yes, only this comment needs to be addressed